pkg/types/set.go

```diff
@@ -148,6 +148,11 @@ func (ts *tsafeSet) Contains(value string) (exists bool) {
 func (ts *tsafeSet) Equals(other Set) bool {
 	ts.m.RLock()
 	defer ts.m.RUnlock()
+	if _other,ok := other.(*tsafeSet); ok {
```

Comment:
Could you gofmt the code change?
https://travis-ci.com/etcd-io/etcd/jobs/220720906#L575

Reply:
Maybe add a comment on why we are checking if `other` is itself?

Reply:
Sure, I will push a new commit.